truss/templates/server.Dockerfile.jinja

```diff
@@ -89,7 +91,23 @@ COPY ./{{ config.model_module_dir }} /app/model
 
 
 {% block run %}
-    {%- if config.live_reload %}
+    {%- if config.docker_server %}
+RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends \
+        curl nginx python3-pip && \
+        rm -rf /var/lib/apt/lists/*
+COPY ./docker_server_requirements.txt /app/docker_server_requirements.txt
+RUN pip install -r /app/docker_server_requirements.txt --no-cache-dir && rm -rf /root/.cache/pip
+{% set proxy_config_path = "/etc/nginx/conf.d/proxy.conf" %}
```

Comment:
Hm, since these variables are each only used in one place, I don't think these sets add a whole lot. I think I left a comment earlier about moving some of these constants to Python (especially things like the supervisor_server_url) I'd consider doing that or just inlining these, I don't think these sets add a whole lot